module/VuFind/src/VuFind/AjaxHandler/GetRecordCover.php

```diff
@@ -60,17 +63,29 @@ class GetRecordCover extends AbstractBase implements AjaxHandlerInterface
      */
     protected $coverRouter;
 
+    /**
+     * @var PhpRenderer
+     */
+    protected $renderer;
+
+    /**
+     * @var bool
+     */
+    protected $useCoverFallbacksOnFail = false;
+
     /**
      * GetRecordCover constructor.
      *
      * @param RecordLoader $recordLoader Record loader
      * @param CoverRouter  $coverRouter  Cover router
      */
     public function __construct(RecordLoader $recordLoader,
-        CoverRouter $coverRouter
+        CoverRouter $coverRouter, PhpRenderer $renderer, Config $config
     ) {
         $this->recordLoader = $recordLoader;
         $this->coverRouter = $coverRouter;
+        $this->renderer = $renderer;
+        $this->useCoverFallbacksOnFail = $config->Content->useCoverFallbacksOnFail ?? false;
```

Comment:
It might be slightly cleaner to just pass in the useCoverFallbacksOnFail value from the factory, rather than passing in the whole Config object... then this class doesn't need to have a specific dependency on the Config class.